assertj-core/src/test/java/org/assertj/core/api/DefaultAssertionErrorCollector_setAfterAssertionErrorCollected_Test.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2023 the original author or authors.
+ */
+package org.assertj.core.api;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+class DefaultAssertionErrorCollector_setAfterAssertionErrorCollected_Test {
+  private List<String> stringList;
+  private List<Throwable> throwableList;
+  private SoftAssertions softly;
+
+  @BeforeEach
+  void given() {
+    stringList = new ArrayList<>();
+    throwableList = new ArrayList<>();
```

Comment:
Rename to `errorMessages` (more intention revealing) and `throwables` (no need to know that it is a list)

Reply:
For some reason I thought it's more readable when you see `someList.size()` or `someList.isEmpty()`. But I've always noticed that it's too expensive in terms of name lenghts. Thank you!